### PR TITLE
internal: readme cleanup

### PR DESCRIPTION
**Pull request by jadekler**
Also document steps to build and test, incl getting google-play-services jar installed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,15 +10,52 @@
...
 
-## <a name='Overview'>Overview<a/>
+## Building locally
 
-The Google APIs Client Library for Java is a flexible, efficient, and powerful Java client library for accessing any HTTP-based API on the web, not just Google APIs. 
+##### One time setup
+
+```
+mkdir /tmp/foo && cd /tmp/foo
+wget --no-check-certificate https://dl.google.com/dl/android/maven2/com/google/android/gms/play-services-basement/8.3.0/play-services-basement-8.3.0.aar
```
Comment (ejona86) [inline README.md]: Why `--no-check-certificate`?

Comment (jadekler) [reply README.md]: *shrug* Copy pasted what I had locally. I can remove, it's probably more canonical to let users figure that out themselves?

Comment (ejona86) [reply README.md]: Why do you need that locally? That means the TLS certs are broken, and isn't something that should generally be ignored.

Comment (jadekler) [reply README.md]: Removed the line + rebasing into master while we continue the conv offline

**Pull request merged**